packages/odyssey-react/src/components/TextInput/TextInput.module.scss

```diff
@@ -11,10 +11,68 @@
  */
 
 .root {
-  @include input-baseline;
+  @include border-radius(var(--BorderRadius));
+
+  display: block;
+  position: relative;
+  width: 100%;
+  max-width: var(--MaxWidth);
+  margin-block: 0;
+  margin-inline: 0;
```

Comment:
these types of styles (margin/padding) were part of the request options to theme. We can update and set them up as component only variables so something like this:

```
// theme module
SpaceMarginBlock: 0,
SpaceMarginInline: 0,

// style module
margin-block: var(--SpaceMarginBlock);
margin-inline: var(--SpaceMarginInline);
```